File: tests/fixtures/java/06-mixed-comments/expected.java
class OrderService {
    /**
     * Procesa el pedido y cobra el monto correspondiente.
     */
    int processOrder(Order order) {
        // Nota: el proveedor de pagos cobra en centavos, no en unidades monetarias completas.
        int amountInCents = (int) (order.getTotal() * 100);

        if (amountInCents <= 0) {
            throw new IllegalArgumentException();
        }

        // Este límite viene de una restricción del banco emisor, no es arbitrario.
        if (amountInCents > 5_000_000) {
            throw new IllegalArgumentException();
        }

        return charge(amountInCents);
    }
}

File: tests/fixtures/csharp/06-mixed-comments/expected.cs
class OrderService
{
    int ProcessOrder(Order order)
    {
        // Nota: el proveedor de pagos cobra en centavos, no en unidades monetarias completas.
        var amountInCents = (int)(order.Total * 100);

        if (amountInCents <= 0)
        {
            throw new ArgumentException();
        }

        // Este límite viene de una restricción del banco emisor, no es arbitrario.
        if (amountInCents > 5_000_000)
        {
            throw new ArgumentException();
        }

        return Charge(amountInCents);
    }
}
